packages/components/src/components/PipelineRuns/PipelineRuns.scss

```diff
@@ -17,6 +17,7 @@ limitations under the License.
   .status-icon {
     margin-right: 0.5em;
     vertical-align: sub;
+    float: left;
```

Comment:
I mentioned in issue 840 that this css is not needed if the the div wrapper is removed from the icon. This is covered in PR #835 https://github.com/tektoncd/dashboard/blob/0c73c4086cbf9f54ede001c4fa122007a0caac1e/packages/components/src/components/PipelineRuns/PipelineRuns.js#L136

Reply:
@carlos-logro see my review comment, using flexbox on the container gives us a cleaner layout. If we could combine the 2 approaches (removing the container, and applying flexbox on a higher level container) that would be great.

Reply:
@AlanGreene I made the change you suggested - this fixes everything nicely for the PipelineRun view but messes up the TaskRun view to look like this:
![Screenshot 2019-12-16 at 14 03 53](https://user-images.githubusercontent.com/52741262/70913062-32f80480-200d-11ea-8396-2a7297009a14.png)
Not sure why this is happening.... suggestions?

Reply:
You'll need to stop the icon from shrinking when other content in the flex container tries to steal space:

```
.definition .status {
  display: flex;

  .status-icon {
    flex-shrink: 0;
  }
}
```

Reply:
thanks, made that change now :)